datasets/gov/nvsc/uzkreciamos_ligos/atvejai.csv

```diff
@@ -0,0 +1,123 @@
+id,dataset,resource,base,model,property,type,ref,source,prepare,level,access,uri,title,description
+,datasets/gov/nvsc/uzkreciamos_ligos/atvejai,,,,,,,,,,,,Nacionalinio visuomenės sveikatos centro užkrečiamų ligų duomenys,
+,,,,,,,,,,,,,,
+,,,,Bendrieji,,,,,,3,,,Užkrečiamųjų ligų atvejai bendrieji,
+,,,,,centras,string,,,,2,open,,NVSC departamenas,
+,,,,,registravimo_vieta,string,,,,2,open,,NVSC skyrius,
+,,,,,miestas,string,,,,2,open,,Miestas,"Miestas, kurioje yra registruotas pacientas, pavadinimas"
+,,,,,galutines_diagnozes_kodas,ref,/datasets/gov/vlk/tlk10am/Tlk10am[code],,,4,open,,Galutinės diagnozės kodas,Pagal TLK-10-AM
+,,,,,galutine_diagnoze,string,,,,4,open,,Galutinė diagnozė,

```

Comment:
ar tai ne `ref` tipo laukas?

Reply:
Jungimas per nepirminį raktą buvo aprašytas pagal dokumentaciją: [nuoroda](https://atviriduomenys.readthedocs.io/dsa/ref.html?highlight=jungimas%20per%20nepirmin%C4%AF#jungimas-per-nepirmini-rakta)

Reply:
Galutinė diagnozė atitinka tarptautinį ligų klasifikatorių, tačiau pats diagnozės pavadinimas nėra patikimas raktas jungimui. Todėl manau, kad čia tinka 4 brandos lygis - atitinka klasifikaciją, bet ref žymėti nėra naudos, nes geriau jungti per kodą, o ne pavadinimą. Ar siūlote kažkaip kitaip žymėti?

Reply:
jeigu duomenys ateina iš to pačio VLK rinkinio, duomenų tipas turėtų būti ref

Reply:
Duomenys atkeliauja iš NVSC ir gaunami iš formų: [formos](https://nvsc.lrv.lt/lt/uzkreciamuju-ligu-valdymas/ataskaitu-pranesimu-formos-ir-aktuali-informacija-aspi/kitos-formos-teikiamos-nvsc). Klasifikatorius yra ne VLK (nors prižiūri vertimą į lietuvių kalbą), o ICD-10 (TLK-10)  (pasaulinis standartas). Atvirų duomenų saugykla  nepalaiko automatinio priskirimo, tą turime padaryti patys, o šiuo atveju naudos vartotojui nėra. Nebent jau atsirado naujas funkcionalumas ir galite užtikrinti, kad atsiradus naujam diagnozės kodui, sistema rodys duomenis, o nepaliks tuščias? Paliktos tuščios reikšmės sugadins analizes visiems visuomenės sveikata besidomintiems vartotojams, diagnozė - vienas svarbiausių požymių.

Reply:
Jeigu turite pasiūlymų dėl Saugyklos tobulinimų/ nusiskundimų ir jie anksčiau nėra pateikti, visuomet kviečiame juos registruoti https://github.com/atviriduomenys/spinta/issues